### PR TITLE
layout_main_screen

### PR DESCRIPTION
**Pull request by lenivecc**

### CODE REVIEW

File: UI/index.html [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+<body>
+    <header>
+
+        <div class="mid-header">
+            <img id="big-main-logo" src="img/Just Chat.svg" alt="Main Logo" style="height: 45px;">
+            <img id="little-main-logo" src="img/JC.svg" alt="Main Logo" style="height: 45px;">
+        </div>
+
+        <div class="left-header">
+            <span style="color: #fff; margin: 0 10px 0 0;" >Filters:</span>
```
Comment (EvgeniZhibrik) [inline UI/index.html:19]: pls, avoid inline styles

File: UI/style.css [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+    margin-left: 30px;
+}
+
+
+
+
+.input-name{
+    display: none;
+    position: absolute;
+    left: -25px;
```
Comment (EvgeniZhibrik) [inline UI/style.css:73]: Pls, try to use rem

File: UI/style.css [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+    z-index: 100;
+    border: 0;
+    border-radius: 5px;
+    padding: 3px;
+    width: 120px;
+}
+
+#label-text:checked~.input-text{
+    display: block;
+}
```
Comment (EvgeniZhibrik) [inline UI/style.css:133]: all filters are almost identical. You can use one class for all checkboxes and one class for all inputs

File: UI/style.css [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+    width: 40px;
+    border-radius: 50%;
+    background: #fff;
+    margin: 0 10px;
+    padding: 5px;
+    font-size: 1.4em;
+    text-align: center;
+}
+
+.UserName{
```
Comment (EvgeniZhibrik) [inline UI/style.css:153]: pls, use lower case words with dashes for classes

Comment (lenivecc) [reply UI/style.css:153]: pls, check my new PR

**Pull request merged**